walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -22,4 +22,4 @@
 
 #random-image-container img {
   margin-top: 20px;
-}
+}
```

Comment:
I assume this was an unintentional change. I would "revert" changes to this file as a whole.